docs/02-app/01-building-your-application/01-routing/04-loading-ui-and-streaming.mdx

```diff
@@ -171,7 +171,7 @@ For more Suspense examples and use cases, please see the [React Documentation](h
 ### SEO
 
 - Next.js will wait for data fetching inside [`generateMetadata`](/docs/app/api-reference/functions/generate-metadata) to complete before streaming UI to the client. This guarantees the first part of a streamed response includes `<head>` tags.
-- Since streaming is server-rendered, it does not impact SEO. You can use the [Mobile Friendly Test](https://search.google.com/test/mobile-friendly) tool from Google to see how your page appears to Google's web crawlers and view the serialized HTML ([source](https://web.dev/rendering-on-the-web/#seo-considerations)).
+- Since streaming is server-rendered, it does not impact SEO. You can use the [Rich Results Test](https://search.google.com/test/rich-results) tool from Google to see how your page appears to Google's web crawlers and view the serialized HTML ([source](https://web.dev/rendering-on-the-web/#seo-considerations)).
```

Comment:
I would change the sentence as well to be honest, "it does not impact SEO" looks like a false statement, there is an impact: https://github.com/vercel/next.js/discussions/50829#discussioncomment-8364251.